src/main/java/net/dv8tion/jda/internal/requests/Requester.java

```diff
@@ -264,17 +275,12 @@ private void applyBody(Request<?> apiRequest, okhttp3.Request.Builder builder)
         builder.method(method, body);
     }
 
-    private void applyHeaders(Request<?> apiRequest, okhttp3.Request.Builder builder, boolean authorized)
+    private void applyHeaders(Request<?> apiRequest, okhttp3.Request.Builder builder)
     {
-        builder.header("user-agent", USER_AGENT)
+        builder.header("user-agent", userAgent)
                .header("accept-encoding", "gzip")
                .header("x-ratelimit-precision", "millisecond"); // still sending this in case of regressions
 
-        //adding token to all requests to the discord api or cdn pages
-        //we can check for startsWith(DISCORD_API_PREFIX) because the cdn endpoints don't need any kind of authorization
-        if (authorized)
-            builder.header("authorization", authConfig.getToken());
-
```

Comment:
Am I missing something? Where did this go?

Reply:
We generally don't use the requester with CDN URLs, which makes this kind of useless. Pretty sure `authorized` is always true.